### PR TITLE
 feat: Add Session type in SessionTrackActivity (#2336)

### LINKED ISSUE #2336: No mention of type of session in SessionTrackActivity
**Actual Behaviour**

The user has no information about the type of session that the speaker will hold whether its a workshop or panel discussion or exhibition etc.

**Expected Behaviour**

The type of the session should be displayed

**Steps to reproduce it**

Go to the speakers section and select any one speaker and see his sessions.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by AkshayCHD**
Fixes #2336 

Changes: Now the session type information is also displayed with other information about the event.
screenshot_1519657266

### CODE REVIEW

**Comment by AkshayCHD** [comment]
@iamareebjamal please review

**Review by iamareebjamal — CHANGES_REQUESTED** [review]
1. Several formatting issues
2. Please don't use same UI shown for track, this will confuse the user, use subtitle instead

**Comment by AkshayCHD** [comment]
@iamareebjamal what do you mean by use subtitle instead, should i display the information in subtitle textview,
can using a separate drawable before the Session Type textview serve as a solution

**Comment by iamareebjamal** [comment]
`should i display the information in subtitle textview`
Yes

`can using a separate drawable before the Session Type textview serve as a solution`
Same as the current solution, not recommended. How if user gonna know what is track and type, what is clickable and what is not

**Comment by AkshayCHD** [comment]
@iamareebjamal please review

**Comment by iamareebjamal** [comment]
Please post screenshots for any UI changes

**Comment by AkshayCHD** [comment]
@iamareebjamal screenshots have been updated

**Comment by iamareebjamal** [comment]
Don't write Type: Talk

Add a subtitle in card header

**Comment by AkshayCHD** [comment]
@iamareebjamal  there already is a session subtitle in the carview and a field for it, and already a field for it.
I have removed Type: from UI please review

**Comment by iamareebjamal** [comment]
Screenshots are not updated

**Comment by AkshayCHD** [comment]
@iamareebjamal screenshots updated

**Review by iamareebjamal — CHANGES_REQUESTED** [review]
Why can't session type be shown as a card header subtitle?

**Comment by AkshayCHD** [comment]
@iamareebjamal i found no subtitle header for card view

**Comment by iamareebjamal** [comment]
Because you have to add it. It is not present already

**Comment by AkshayCHD** [comment]
@iamareebjamal please review, screenshot updated

**Comment by iamareebjamal** [comment]
The subtitle should be in card header

**Comment by AkshayCHD** [comment]
@iamareebjamal please review, screenshot updated

**Comment by iamareebjamal** [comment]
The subtitle should be in new line rather than in the same line aligning to the right and its font size should be smaller

**Comment by AkshayCHD** [comment]
@iamareebjamal please review

**Comment by iamareebjamal** [comment]
The alignment of subheader with title is off

**Comment by AkshayCHD** [comment]
@iamareebjamal please review now, screenshots updated

File: android/app/src/main/res/layout/tracksactvity_item.xml [hunk 1]
```diff
@@ -24,29 +24,42 @@
...
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:layout_gravity="end"
-                    android:background="?attr/selectableItemBackgroundBorderless"
-                    android:contentDescription="@string/session_bookmark_status"
                     android:padding="@dimen/padding_medium"
-                    app:srcCompat="@drawable/ic_bookmark_border_white_24dp" />
-
+                    android:textSize="@dimen/text_size_large"
+                    tools:text="Type" />
```
Comment (iamareebjamal) [inline android/app/src/main/res/layout/tracksactvity_item.xml]: Why is it large? Also the padding around the item is huge. Please make it look standard yourself first

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

**Comment by AkshayCHD** [comment]
@iamareebjamal please review

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**